pkg/testutils/testutils.go

```diff
@@ -0,0 +1,227 @@
+package testutils
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	configv1 "github.com/openshift/api/config/v1"
+	cloudingressv1alpha1 "github.com/openshift/cloud-ingress-operator/pkg/apis/cloudingress/v1alpha1"
+
+	awsclient "github.com/openshift/cloud-ingress-operator/pkg/awsclient/mock"
+	machineapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/utils/pointer"
+	awsprovider "sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsproviderconfig/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+const masterMachineLabel string = "machine.openshift.io/cluster-api-machine-role"
+
+const DefaultRegionName string = "us-east-1"
+const DefaultAzName string = "us-east-1a"
+const DefaultAPIEndpoint string = "https://api.unit.test:6443"
+const DefaultClusterDomain string = "unit.test"
+
+// ClusterTokenId represents the part of identifiers which is varied by the installer, eg
+// clustername-clustertokenid, as in load balancer names: foo-12345-us-east-1a
+const ClusterTokenId string = "12345"
```

Comment:
Curious why this doesn't use the grouping syntax
```
const (
    ...
)
```
...though I see it's just a straight cut/paste from the original.

Reply:
If I'm honest, it started out as one constant, then another popped up, and so I added it as another `const` line and didn't think about the grouping.

I'm happy to change the formatting, especially if it's better Go practice - I'm not entirely deep into the Go culture (yet).

Reply:
Then I am happy to leave it for Future People to change if they so desire.